.github/workflows/ci.yaml

```diff
@@ -26,3 +26,7 @@ jobs:
         bundler-cache: true
     - name: Run tests
       run: bundle exec rake
+    - name: Report code coverage
+      if: "${{ github.ref == 'refs/heads/master' && matrix.ruby-version == '3.2' }}"
+      continue-on-error: true
+      uses: paambaati/codeclimate-action@v5
```

Comment:
This won't work without the secret.  Looking at another gem, add this above the steps

```
    env:
      CC_TEST_REPORTER_ID: "${{ secrets.CC_TEST_REPORTER_ID }}"
```

Example: https://github.com/ManageIQ/linux_admin/blob/746d84d93827ef930f31ee7611e93d56c8c83e84/.github/workflows/ci.yaml#L19-L20